controllers/NgoController.js

```diff
@@ -57,17 +61,28 @@ exports.export = [
   },
 ];
 
-exports.status = [
-  function (req, res) {
-    const query = [{ $group: { _id: "$status", status: { $sum: 1 } } }];
-    handleStatus(res, query, Ngo);
-  },
-];
-
 // Verify the NGO Registration details
 exports.updateStatus = [
   auth,
   function (req, res) {
+    // Todo: check email verification and create user
     handleStatusUpdate(req, res, Ngo);
   },
 ];
+
+exports.verifyEmail = [
+  asyncH(async (req, res) => {
+    // req.body.email_verified = false
+    const ngo = await Ngo.findOneAndUpdate(
+      req.body,
```

Comment:
better to form a query object with the fields you need than using the body directly.

Reply:
As discussed, the approach is client will receive the details and call the API via PUT action.